src/main/java/roomescape/entity/MemberRole.java

```diff
@@ -0,0 +1,11 @@
+package roomescape.entity;
+
+public enum MemberRole {
+    ADMIN(),
+    USER(),
```

Comment:
```suggestion
    ADMIN,
    USER,
```

멤버변수가 없는 경우 위와 같이 작성할 수 있습니다~

Reply:
넵 감사합니다 수정하였습니다 👍